CONTRIBUTING.md

```diff
@@ -1,81 +1,106 @@
-using your favorite environment manager).
-
-Then build the pages:
-
-```shell
-$ starterkit_ci build --allow-warnings
-$ starterkit_ci check --allow-warnings
-```
-
-and start a web server to host them:
-
-```shell
-$ cd build
-$ python -m http.server 8000
-```
-You can see your local version by using a web-browser to navigate to `http://localhost:8000` or wherever it says it's serving the book.
-
-## Contributing notebooks
-
-Notebooks should contain no output to help keep the size of this repository manageable.
-The output for each cell will be filled automatically as part of the deployment process.
-
-[conduct]: CONDUCT.md
-[repo]: https://github.com/lhcb/starterkit-lessons
-[issues]: https://github.com/lhcb/starterkit-lessons/issues
-[license]: LICENSE.md
-[pro-git-chapter]: https://git-scm.com/book/en/v2/GitHub-Contributing-to-a-Project
-[gh-fork]: https://help.github.com/en/articles/fork-a-repo
-[gh-pull]: https://help.github.com/en/articles/about-pull-requests
-[gh-fork-pull]: https://reflectoring.io/github-fork-and-pull/
-
-
-```eval_rst
-.. toctree::
-    :hidden:
-
-    CONDUCT.md
-    LICENSE.md
-```
+[HSF training][hsf-training] is an open source project,
+and we welcome contributions of all kinds:
+new lessons,
+fixes to existing material,
+bug reports,
+and reviews of proposed changes are all welcome.
+
+## Contributor Agreement
+
+By contributing,
+you agree that we may redistribute your work under [our license](LICENSE.md).
+In exchange,
+we will address your issues and/or assess your change proposal as promptly as we can,
+and help you become a member of our community.
+Everyone involved in [HSF training][hsf-training]
+agrees to abide by our [code of conduct](CODE_OF_CONDUCT.md).
+
+## How to Contribute
+
+The easiest way to get started is to file an issue
+to tell us about a mistake,
+some awkward wording,
+a factual error.
+This is a good way to introduce yourself
+and to meet some of our community members.
+
+1. If you do not have a [GitHub][github] account,
+    you can write the convenors an email.
+    However,
+    we will be able to respond more quickly if you use one of the other methods described below.
+
+2. If you have a [GitHub][github] account,
+    or are willing to [create one][github-join],
+    but do not know how to use Git,
+    you can report problems or suggest improvements by [creating an issue][issues].
+    This allows us to assign the item to someone
+    and to respond to it in a threaded discussion.
+
+3. If you are comfortable with Git,
+    and would like to add or change material,
+    you can submit a pull request (PR).
+    Instructions for doing this are [included below](#using-github).
+
+## What to Contribute
+
+There are many ways to contribute,
+from writing new exercises and improving existing ones
+to updating or filling in the documentation
+and submitting [bug reports][issues]
+about things that do not work, aren not clear, or are missing.
```

Comment:
```suggestion
about things that do not work, are not clear, or are missing.
```

Reply:
ah shoot! Thanks for pointing this out!

Let's merge this anyway as-is and this will just be updated the next time I sync everything